roles/new-do-droplet/defaults/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+admin_username: admin
+do_droplet_size: s-1vcpu-1gb
+do_droplet_region: SFO3
+do_droplet_image: ubuntu-22-10-x64
+do_droplet_enable_monitoring: true
+do_droplet_enable_backups: false
```

Comment:
I think it makes sense to turn backups on by default? I think they are turned on for most of our droplets already. I know they cost extra but since we have no other backup policy I'd rather err on the side of backing up too much.